lib/estree-to-babel.js

```diff
@@ -53,3 +55,33 @@ module.exports = (node) => {
 function setObjectProperty(node) {
     node.type = 'ObjectProperty';
 }
+
+function setComments(node) {
+    const {comments} = node;
+    
+    if (!comments)
+        return;
+    
+    delete node.comments;
+    node.leadingComments = undefined;
+    node.trailingComments = undefined;
+    node.innerComments = undefined;
+    
+    for (const comment of comments) {
+        // Default to leading comment.
+        const type = comment.trailing ? 'trailingComments' : !comment.leading ? 'innerComments' : 'leadingComments';
```

Comment:
Let's create a function in the bottom of a file:

```js
 function getCommentType({trailing, leading}) {
     if (trailing)
        return 'trailingComments';
      
    if (comment.leading)
        return 'leadingComments';
               
    return 'innerComments' ;
}
```
And use it:

```suggestion       
        const type = getCommentType(comment);
```

Also let's move out function `setComment` into file `comments.js`.